src/Sulu/Bundle/DocumentManagerBundle/Tests/app/Resources/pages/default.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" ?>
+
+<template xmlns="http://schemas.sulu.io/template/template"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/template-1.0.xsd"
+          >
+
+    <key>default</key>
+
+    <view>ClientWebsiteBundle:templates:animals</view>
+    <controller>SuluWebsiteBundle:Default:index</controller>
+    <cacheLifetime>2400</cacheLifetime>
+
+    <meta>
+        <title lang="de">Tiers</title>
+        <title lang="en">Animals</title>
+    </meta>
+
+    <properties>
+        <property name="title" type="text_line" mandatory="true">
+            <meta>
+                <title lang="de">Tier</title>
+                <title lang="en">Animals</title>
+            </meta>
+
+            <tag name="sulu.rlp.part"/>
+        </property>
+
+        <property name="url" type="resource_locator" mandatory="true">
+            <meta>
```

Comment:
Necessary to add these bootstrap files currently unless we refactor even more stuff. But perhaps we could remove th efunctional test from this bundle.